### ISSUE TITLE
Black Screen Overlay

### ISSUE THREAD

**Issue by Alpha-Obeisance** [gerritdevriese/kzones#57]
When moving a window, upon overlay presentation; the screen goes black. I've never had any  display issues; so I'm not sure if there's a settting I've failed to enable; or if I've failed to install some kind of dependency. This is probably my favorite script for KDE Plasma in regards to window management, but this is a bit jarring to deal with anytime I feel like dragging a window.

Apologies if this is not a bug, and instead is merely caused by my own ignorance. 

Before: 
![image](https://github.com/gerritdevriese/kzones/assets/90144672/eb7da957-effa-4a73-b9ee-b1224c99092c)

During window drag
![Screenshot_20240103_000007](https://github.com/gerritdevriese/kzones/assets/90144672/50d4fa70-d56b-4964-8ce3-90d51a90ab10)

I use X11.

**Comment by gerritdevriese** [comment]
Can you check if your compositor is enabled:

Plasma settings: Display and Monitor - Compositor

**Comment by gerritdevriese** [comment]
Hi,

I'm pretty sure that disabling the compositor causes this issue, as it is needed for Plasma to be able to draw transparency.
Can you try if it works when you enable it? Is there a particular reason you want it disabled?

Nice rice BTW :)

**Comment by Alpha-Obeisance** [comment]
You would be correct! I know very little about the compositor; and honestly don't recall what caused me to make habit of disabling it to be honest haha. I apologize for bothering you with this issue; but I thank you for educating me on this! 

Thanks for the compliments on the setup too =). Ironically so, I lost my tilix background when I reinstalled after posting this issue and I can't find that picture of Jinx anywhere lol! 

Many thanks for this great script. It's a must have every time I set up!
